app/services/cycle_timetable.rb

```diff
@@ -204,6 +209,11 @@ def self.fake_schedules
         next_year => {
           find_opens: 6.days.from_now,
           apply_opens: 7.days.from_now,
+          show_deadline_banner: 9.days.from_now,
+          apply_1_deadline: 10.days.from_now,
+          apply_2_deadline: 11.days.from_now,
+          reject_by_default: 12.days.from_now,
+          find_closes: 13.days.from_now,
```

Comment:
Do we need these for the `previous_cycle` case as well?

Reply:
I don't think we use the previous years dates so i think we're okay

Reply:
I think the error on sandbox is because we are hard-coded looking for reject by default for 2021, which is `previous_year`. We should probably change the check to be a bit smarter though